scripts/cluster_common.sh

```diff
@@ -24,9 +24,16 @@ create_k8s_cluster() {
             echo "FATAL: bringing up k8s cluster '${ctx}' not supported"
             exit 255
         }
-        ${cluster_up}
+        . ${cluster_up}
```

Comment:
Why is this needed?

Reply:
I do not like it... Thinking on this now, basically `cluster_up` may set some variables, which require later, at the moment it's KUBECONFIG, do you think this is a case?

Reply:
For my particular case and to be consistent with everything else I can assume that KUBECONFIG is at default ~/.kube/config location and remove this change

Reply:
That makes sense, thanks!